sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs

```diff
@@ -595,6 +596,7 @@ public async Task SetBlobAccessTier_Snapshot()
         }
 
         [Test]
+        [Ignore("service bug")]
```

Comment:
Nit - let's file a bug in our repo to re-enable this

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/13507